pkg/cmd/cli/cmd/importimage.go

```diff
@@ -374,13 +374,15 @@ func (o *ImportImageOptions) createImageImport() (*imageapi.ImageStream, *imagea
 		},
 		Spec: imageapi.ImageStreamImportSpec{Import: true},
 	}
+	insecureAnnotation := stream.Annotations[imageapi.InsecureRepositoryAnnotation]
+	insecure := o.Insecure || insecureAnnotation == "true"
```

Comment:
If a user passes `oc import-image --insecure=false`, should that take priority over the image stream?  Why or why not?